soh/soh/Enhancements/mods.cpp

```diff
@@ -655,7 +655,7 @@ void UpdateDirtPathFixState(int32_t sceneNum) {
 
 void RegisterMenuPathFix() {
     GameInteractor::Instance->RegisterGameHook<GameInteractor::OnTransitionEnd>([](int32_t sceneNum) {
-        UpdateDirtPathFixState(sceneNum); });
+        UpdateDirtPathFixState(sceneNum);
         UpdateDirtPathFixState(sceneNum);
```

Comment:
It looks like this function now has two calls here XD.

Reply:
fixed this in my branch and will remove it next PR update haha. thanks for the call out <3